tensorflow_quantum/python/optimizers/utils_test.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 The TensorFlow Quantum Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Utils for tensorflow quantum optimizers
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import tensorflow as tf
+from absl.testing import parameterized
+import numpy as np
+from .utils import function_factory
+
+
+class RotosolveMinimizerTest(tf.test.TestCase, parameterized.TestCase):
+    """Tests for optimizer utils"""
+
+    def test_function_factory(self):
+        """Test the function_factory"""
+
+        class LinearModel(object):
+            """ A simple tensorflow linear model"""

```

Comment:
period please, and please trim the meaningless white space in front of sentence.

Reply:
Fixed in 564256fa0316a94cc5fc82a6285ebe64285afab8